src/map/lua/lua_baseentity.cpp

```diff
@@ -6644,13 +6645,19 @@ inline int32 CLuaBaseEntity::setEminenceCompleted(lua_State *L)
     CCharEntity* PChar = (CCharEntity*)m_PBaseEntity;
     uint16 recordID = (uint16)lua_tointeger(L, 1);
 
-    bool status;
-    if (lua_gettop(L) < 2)
-        status = true;
+    bool repeat = false;
+    if (lua_gettop(L) > 1)
+        repeat = lua_tointeger(L, 2) != 0;
+
+    bool status = true;
+    if (lua_gettop(L) > 2)
+        status = lua_tointeger(L, 3) != 0;
+
+    if (repeat)
+        charutils::SetEminenceRecordProgress(PChar, recordID, 0);
     else
-        status = lua_tointeger(L, 2) != 0;
+        charutils::DelEminenceRecord(PChar, recordID);
```

Comment:
Y'all got any of them Allman-style braces?

Reply:
Alright, but they're just so damned ugly! </YouManiacsYouBlewItUp>

Reply:
```
Im
{
    ugly?
    ;_;
}
```

well now you hurt Curly';s feelings. Meanie.

Reply:
```c++
    bool status = true;
    if (lua_gettop(L) > 2)
    {
        status = lua_tointeger(L, 3) != 0;
    }

    if (repeat)
    {
        charutils::SetEminenceRecordProgress(PChar, recordID, 0);
    }
    else
    {
        charutils::DelEminenceRecord(PChar, recordID);
    }```

Now they can go surfing on all these waves!

Reply:
![image](https://user-images.githubusercontent.com/6871475/91466853-b8e77300-e85d-11ea-8ca2-028f8b5a5edd.png)

Reply:
the only thing I really;y hate is when someone does half their conditionals one way and have the other:
```
if something
{
    stuff
}
else
    WHY AM I DANGLING
```

Reply:
Maybe we can agree on 

```c++
if something
    { stuff }
else
    { I'm not dangling! }```
:smirk: